ingester-example/src/main/java/io/greptime/bench/benchmark/BatchingWriteBenchmark.java

```diff
@@ -51,21 +53,18 @@ public class BatchingWriteBenchmark {
     private static final Logger LOG = LoggerFactory.getLogger(BatchingWriteBenchmark.class);
 
     public static void main(String[] args) throws Exception {
```

Comment:
The `db_endpoint` and `db_name` in the class comments(L45~L46) can also be removed.